### PR TITLE
Feature/2.3/cloudscale server

### PR DESCRIPTION
**Pull request by gaudenz**
##### ISSUE TYPE
 - New Module Pull Request

##### COMPONENT NAME
cloudscale_server

##### ANSIBLE VERSION
```
ansible 2.3.0 (feature/2.3/cloudscale_server e9acf74eb9) last updated 2017/01/12 13:37:43 (GMT +200)
  config file = /home/gaudenz/.ansible.cfg
  configured module search path = [u'/home/gaudenz/.ansible/plugins/library']
```

##### SUMMARY
This is a cloud module to create, start, stop and delete servers on the cloudscale.ch IaaS platform through their API.

### CODE REVIEW

File: test/integration/targets/cloudscale_server/meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+dependencies:
+  - prepare_tests
```
Comment (resmo) [inline test/integration/targets/cloudscale_server/meta/main.yml:2]: AFAICS this role is missing, would you mind add it to the PR?

**Review by resmo — COMMENTED** [review]
Over all a clean and straight forward module. Well done @gaudenz

File: test/integration/targets/cloudscale_server/meta/main.yml (hunk 1 above)
Comment (gaudenz) [reply test/integration/targets/cloudscale_server/meta/main.yml:2]: @resmo I don't know why this role does not exist, but this dependency is listed in all test roles unter integration/targets so I thought this should be added by default.

**Comment by gaudenz** [comment]
AFAICS the test failure has nothing to do with the changes in this PR. Is there a way to schedule a recheck?

File: test/integration/targets/cloudscale_server/meta/main.yml (hunk 1 above)
Comment (resmo) [reply test/integration/targets/cloudscale_server/meta/main.yml:2]: I see, I am not aware of the purpose but at least it does not hurt. Ok then

**Comment by resmo** [comment]
The only way you can do it is with force push IMHO

**Comment by resmo** [comment]
I kicked the build again, all fine. 

shipit

**Pull request merged**